tests/phpunit/tests/blocks/serialize.php

```diff
@@ -94,8 +93,8 @@ public function test_traverse_and_serialize_identity_from_parsed( $original ) {
 
 		$actual = traverse_and_serialize_blocks(
 			$blocks,
-			function ( $block ) {
-				return $block;
+			function () {
```

Comment:
It's fine to omit this param now that it is optional.

Reply:
Yeah, but don't we have to omit the entire test then? We already have one for `traverse_and_serialize_blocks` without a callback, don't we?

Reply:
This test mirrors `serialize_blocks` to ensure that they work the same way.